.github/workflows/pr-auto-label.yml

```diff
@@ -12,11 +12,12 @@ permissions:
     checks: write # to generate status
 
 jobs:
-  remove-old-labels:
-    name: Remove old PR labels
+  changelog-automation:
+    name: Changelog automation
     runs-on: ubuntu-latest
     steps:
```

Comment:
Let's add this condition to skip all these steps when PR is raised by dependabot

```
runs-on: ubuntu-latest
if: github.actor != 'dependabot[bot]'
steps:
```

Reply:
Im not sure if this will have the effect that we want 🤔

A skipped job isn't considered successful, so we will keep blocking dependabot PRs this way 😬

Reply:
Oh I see. Any idea if we use [exit with success status](https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions#about-exit-codes), will this work?

Reply:
Yeah, we can probably do something similar :)